Gemfile.lock

```diff
@@ -807,7 +824,7 @@ DEPENDENCIES
   spring-commands-rspec (= 1.0.4)
   test_after_commit (= 0.4.1)
   timecop (= 0.7.3)
-  twitter (= 4.8.1)
+  twitter (= 5.14.0)
   twitter-text (= 1.11.0)
   typhoeus (= 0.7.1)
   uglifier (= 2.7.1)
```

Comment:
Please commit `Gemfile.lock` changes after running `bin/spring stop; DB=mysql bundle` for now.